contracts/script/IncredibleSquaringDeployer.s.sol

```diff
@@ -73,396 +76,55 @@ contract IncredibleSquaringDeployer is Script, Utils {
-                quorumsStrategyParams[i] = new IStakeRegistry.StrategyParams[](
-                    numStrategies
-                );
-                for (uint j = 0; j < numStrategies; j++) {
-                    quorumsStrategyParams[i][j] = IStakeRegistry
-                        .StrategyParams({
-                            strategy: deployedStrategyArray[j],
-                            // setting this to 1 ether since the divisor is also 1 ether
-                            // therefore this allows an operator to register with even just 1 token
-                            // see https://github.com/Layr-Labs/eigenlayer-middleware/blob/m2-mainnet/src/StakeRegistry.sol#L484
-                            //    weight += uint96(sharesAmount * strategyAndMultiplier.multiplier / WEIGHTING_DIVISOR);
-                            multiplier: 1 ether
-                        });
-                }
-            }
-            incredibleSquaringProxyAdmin.upgradeAndCall(
-                TransparentUpgradeableProxy(
-                    payable(address(registryCoordinator))
-                ),
-                address(registryCoordinatorImplementation),
-                abi.encodeWithSelector(
-                    regcoord.RegistryCoordinator.initialize.selector,
-                    // we set churnApprover and ejector to communityMultisig because we don't need them
-                    incredibleSquaringCommunityMultisig,
-                    incredibleSquaringCommunityMultisig,
-                    incredibleSquaringCommunityMultisig,
-                    incredibleSquaringPauserReg,
-                    0, // 0 initialPausedStatus means everything unpaused
-                    quorumsOperatorSetParams,
-                    quorumsMinimumStake,
-                    quorumsStrategyParams
-                )
-            );
-        }
-
-        incredibleSquaringServiceManagerImplementation = new IncredibleSquaringServiceManager(
-            avsDirectory,
-            registryCoordinator,
-            stakeRegistry,
-            incredibleSquaringTaskManager
-        );
-        // Third, upgrade the proxy contracts to use the correct implementation contracts and initialize them.
-        incredibleSquaringProxyAdmin.upgrade(
-            TransparentUpgradeableProxy(
-                payable(address(incredibleSquaringServiceManager))
-            ),
-            address(incredibleSquaringServiceManagerImplementation)
-        );
+    function run() external {
+        // Eigenlayer contracts
+        vm.startBroadcast(deployer);
+        AGGREGATOR_ADDR = vm.envAddress("AGGREGATOR_ADDR");
+        TASK_GENERATOR_ADDR = vm.envAddress("TASK_GENERATOR_ADDR");
+        CONTRACTS_REGISTRY_ADDR = vm.envAddress("CONTRACTS_REGISTRY_ADDR");
+        OPERATOR_ADDR = vm.envAddress("OPERATOR_ADDR");
+        contractsRegistry = ContractsRegistry(CONTRACTS_REGISTRY_ADDR);
+        deployer = vm.rememberKey(vm.envUint("PRIVATE_KEY"));
```

Comment:
setup should run before `run()` so this can be removed along with the label below